packages/server/test/unit/config_spec.coffee

```diff
@@ -322,9 +322,18 @@ describe "lib/config", ->
           @setup({testFiles: "**/*.coffee"})
           @expectValidationPasses()
 
-        it "fails if not a string", ->
+        it "passes if an array of strings", ->
+          @setup({ignoreTestFiles: ["**/*.coffee", "**/*.jsx"]})
```

Comment:
you mean `testFiles` here right ;)

Reply:
Yup, let me change these two places within `config_spec`